src/main/java/com/pmn/gmt/domain/teacher/service/FindTeachersByNameFilterService.java

```diff
@@ -0,0 +1,10 @@
+package com.pmn.gmt.domain.teacher.service;
+
+import com.pmn.gmt.domain.teacher.presentation.data.dto.NameDto;
+import com.pmn.gmt.domain.teacher.presentation.data.dto.TeacherDto;
+
+import java.util.List;
+
+public interface FindTeachersByNameFilterService {
```

Comment:
```suggestion
public interface FindTeachersByNameService {
```

선생님을 이름으로 찾는 것이니 서비스의 이름을 FindTeachersByNameFilterService로 짓는 것 보다는  FindTeachersByNameService로 하는 게 나아보여요!

Reply:
수정했습니다 ㅎㅎ